### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by worldbostin** [Jay-study-nildana/FrontEndForStudents#33] [General]
I am new on here from Edureka.

**Comment by Jay-study-nildana** [comment]
@worldbostin 

Hello and welcome. 

There are many things you can do here. First thing is to check out my YouTube Channel which has many beginner videos. 

https://www.youtube.com/channel/UCJJVulg4J7POMdX0veuacXw

Also, see the many questions other students have asked on this community. 

and you have more questions, you can ask me here. You can also join me on discord and chat and even fix zoom calls if you detailed questions,.

Thanks.

Jay (Vijayasimha BR)
Coding Tutor. Career and Life Coach
Discord: Jay_HisManyHobbies#6392
